src/pointfree/toPromise.spec.js

```diff
@@ -0,0 +1,39 @@
+const test = require('tape')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+
+const isFunction  = require('../core/isFunction')
+
+const Async = require('../Async')
+const toPromise = require('./toPromise')
+
+test('toPromise pointfree', t => {
+  t.plan(13)
```

Comment:
Usually for pointfree functions, we only test if the underlying function on the container has been called using `sinon`. If I am not wrong, functionality tests are to be reserved for the crocks themselves.

@evilsoft - Thoughts?